crates/chain/src/tx_graph.rs

```diff
@@ -1214,7 +1214,10 @@ impl<A> Default for ChangeSet<A> {
 impl<A> ChangeSet<A> {
     /// Returns true if the [`ChangeSet`] is empty (no transactions or txouts).
     pub fn is_empty(&self) -> bool {
-        self.txs.is_empty() && self.txouts.is_empty()
+        self.txs.is_empty()
+            && self.txouts.is_empty()
+            && self.anchors.is_empty()
+            && self.last_seen.is_empty()
     }
```

Comment:
`tx_graph::ChangeSet` implements `Append` and `Append` already has a method `is_empty`. So this is a duplicate and we can just remove it.

The caveat (which is not a big one imo), is the caller needs to `use bdk_chain::Append` to call it.